src/main/java/com/fruits/ecommerce/services/Interfaces/UserService.java

```diff
@@ -26,7 +25,7 @@ public interface IUserService {
      * //@throws InvalidRoleException If the specified role is invalid.
      * //@throws InvalidUserDataException If the user data is invalid or incomplete.
      */
-    UserDTO register(UserDTO userDTO);
+//    UserDTO register(UserDTO userDTO);
```

Comment:
Let's keep code base clean, no need for commented code

Reply:
You are right; I will change the code as per the improvement you suggested.